numpyro/contrib/einstein/kernels.py

```diff
@@ -412,14 +425,16 @@ def kernel(x, y):
 
 class GraphicalKernel(SteinKernel):
     """
-    Calculates graphical kernel :math: `k(x,y) = diag({K^(l)(x,y)}_l)` from [1].
+    Calculates graphical kernel :math:`k(x,y) = diag({K^(l)(x,y)}_l)` (TODO: correct this formula) from [1].
```

Comment:
@OlaRonning Could you help me correct this formula? I couldn't find this kernel in the reference.

Reply:
@fehiepsi the corresponds is established (page 4 last paragraph before section 3.3) in 
```
     2. *Stein Variational Gradient Descent with Matrix-Valued Kernels* by Wang, Tang, Bajaj, and Liu
```

Reply:
I see, I think we need to modify :math:`k(x,y) = diag({K^(l)(x,y)}_l)` to `k(x,y) = diag(K_l(x,y))` (the current one does not rendered correctly so I'm not sure how to rewrite the formula into the one that you prefer? Maybe we need to say what is K and l too.

Reply:
I think I should make the comment clearer. In latex, `K^(l)` will not be the same as K^l

Reply:
`k(x,y) = diag(K_l(x,y))` is good. `K^(l)` should be `K^{(l)}` I suppose.

Reply:
How about:
``` python
    """
    Calculates graphical kernel :math:`k(x,y) = diag({K_l(x_l,y_l)})` for local kernels :math:`K_l` from [1][2].

    **References:**

    1. *Stein Variational Message Passing for Continuous Graphical Models* by Wang, Zheng, and Liu
    2. *Stein Variational Gradient Descent with Matrix-Valued Kernels* by Wang, Tang, Bajaj, and Liu

    :param local_kernel_fns: A mapping between parameters and a choice of kernel
        function for that parameter (default to default_kernel_fn for each parameter)
    :param default_kernel_fn: The default choice of kernel function when none is
        specified for a particular parameter
    """
```

Reply:
Done, thanks for suggesting!